--- TownCrier.sol	2018-04-17 16:50:03.120501193 -0400
+++ TownCrier-rinkeby.sol	2018-04-17 16:50:24.756583848 -0400
@@ -15,8 +15,8 @@
     event DeliverInfo(uint64 requestId, uint fee, uint gasPrice, uint gasLeft, uint callbackGas, bytes32 paramsHash, uint64 error, bytes32 respData); // log of responses
     event Cancel(uint64 requestId, address canceller, address requester, uint fee, int flag); // log of cancellations
 
-    // address of the SGX account (mainnet)
-    address public constant SGX_ADDRESS = 0x18513702cCd928F2A3eb63d900aDf03c9cc81593;
+    // address of the SGX (Rinkeby)
+    address public constant SGX_ADDRESS = 0x3a8de03f19c7C4c139B171978F87BFAC9ffe99c0;
 
     uint public GAS_PRICE = 5 * 10**10;
     uint public MIN_FEE = 30000 * GAS_PRICE; // minimum fee required for the requester to pay such that SGX could call deliver() to send a response
